lib/contextual_scaffold.dart

```diff
@@ -75,38 +76,25 @@ class ContextualScaffold<T> extends StatelessWidget {
       builder: (BuildContext context, Widget child) {
         bool isActionModeEnable() =>
             Provider.of<ItemsController<T>>(context).actionModeEnable;
-        return Stack(
-          children: [
-            Positioned.fill(
-              child: Scaffold(
-                appBar: appBar,
-                body: child,
-                floatingActionButton: floatingActionButton,
-                floatingActionButtonLocation: floatingActionButtonLocation,
-                floatingActionButtonAnimator: floatingActionButtonAnimator,
-                persistentFooterButtons: persistentFooterButtons,
-                drawer: drawer,
-                endDrawer: endDrawer,
-                bottomNavigationBar: bottomNavigationBar,
-                bottomSheet: bottomSheet,
-                backgroundColor: backgroundColor,
-                resizeToAvoidBottomInset: resizeToAvoidBottomInset,
-                primary: primary,
-                drawerDragStartBehavior: drawerDragStartBehavior,
-                extendBody: extendBody,
-                extendBodyBehindAppBar: extendBody,
-                drawerScrimColor: drawerScrimColor,
-                drawerEdgeDragWidth: drawerEdgeDragWidth,
-              ),
-            ),
-            if (isActionModeEnable())
-              Positioned(
-                top: 0,
-                right: 0,
-                left: 0,
-                child: contextualAppBar,
-              ),
-          ],
+        return Scaffold(
```

Comment:
There was a reason the contextual appbar was stacked. The reason being that you can have multiple contextual appbar. Run the "Whatsapp" sample code to see what i mean here.

Reply:
I see. So to avoid the issue I described, "where `ContextualActionBar` was calling `Scaffold.of(context)` (contextual_action_bar.dart line 216) but since it was a sibling in the `Stack`, it could not find the correct `Scaffold`", it looks like you wrap everything in another `Scaffold`? I wonder if there is a way to avoid wrapping it with another `Scaffold`.